lib/stellar_core_commander/process.rb

```diff
@@ -279,6 +279,11 @@ def manual_close?
 
     Contract None => Bool
     def close_ledger
+      if is_sqlite
+        $stderr.puts "#{idname}: close_ledger is not supported for SQLite DB. Please switch to PostgreSQL."
+        return false
```

Comment:
not sure if `return false` is the right thing here (it's unclear what this return value is for), maybe crashing (raise) in this case is better